README.md

```diff
@@ -0,0 +1,226 @@
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Backend</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org">Ruby on Rails (API-only mode)</a></li>
+  </ul>
+</details>
+
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Authentication</summary>
+  <ul>
+    <li>JWT-based authentication</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Documentation</summary>
+  <ul>
+    <li><a href="https://swagger.io/">Swagger</a></li></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Course Booking:**  Users can browse available courses, specify their city, and choose preferred dates for course attendance..
+
+- **Endpoint Documentation:** Comprehensive API documentation is provided to guide frontend developers on how to interact with the backend.
+
+- **User Authentication:** Secure user authentication and authorization mechanisms are implemented to protect sensitive user data and interactions.
+
+- **Scalability:** Built with scalability in mind, the backend can handle a growing number of users and course offerings.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Link to Frontend<a name="live-demo"></a>
+
+- Check out VECS SCHOOL frontend app [here.](https://github.com/smunoz1988/vecs-school-frontend)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>  
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Ensure you have Ruby installed on your system. You can check by running the code below in your terminal.
+```sh
+  ruby --version
+```
+### Setup
+
+Clone or download the VECS School repository to your local machine.
+```sh
+  git clone https://github.com/christianonoh/vecs-school-backend.git
+```
+Open your terminal and navigate to the cloned project directory.
+
+### Install
+
+This project does not require additional dependencies, just Ruby interpreter. However, it uses Rubocop as a linter and you should set it up as well.
+
+```sh
+bundle install
+```
+**Credentials Setup:** Generate the master key.
+
+- Delete the `config/credentials.yml.enc` file and run:
+
+```sh
+EDITOR="code --wait" rails credentials:edit
+```
+- This will generate new `config/master.key` and `config/credentials.yml.enc` files.
+
+**Database Setup:** Navigate to the project directory and set up the database.
+- In case it's necessary, add your database credentials to the `config/database.yml`.
+- Run the following commands to create a database and run database migrations
+```sh
+rails db:create
+rails db:migrate
+```
```

Comment:
- [ ] You have a very functional `seeds.rb`.  Kindly add to your db commands `rails db:seed`.

That way, your users will enjoy a pre-seeded database. 👍